pkg/controller/apiextensions/composite/reconciler_test.go

```diff
@@ -36,6 +36,13 @@ import (
 	"github.com/crossplane/crossplane/apis/apiextensions/v1beta1"
 )
 
+// Various fields on our types take a *string, but
+// We want to pass them literals or values from constants.
+// Pass them through this first
+func strPtr(x string) *string {
+	return &x
+}
```

Comment:
There's an upstream util for this we could use - https://github.com/kubernetes/utils/blob/master/pointer/pointer.go#L91

Reply:
TIL 😃 Switching over to use that